cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/CommandCommons.java

```diff
@@ -31,6 +31,8 @@ public Integer call() throws Exception {
 
     if (this.verbose) {
       CLILogger.setLevel(Level.DEBUG);
+    } else {
+      CLILogger.setLevel(Level.INFO);
```

Comment:
Why this change? Was there anything not working so far? I thought the default is handled by logback.xml